dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -277,17 +277,20 @@ public GHBlob tryRetrievingBlob(GHRepository repo, String path, String branch)
         return gitHubUtil.tryRetrievingBlob(repo, path, branch);
     }
 
-    public void modifyOnGithub(GHContent content,
+    public boolean modifyOnGithub(GHContent content,
                                String branch, String img, String tag,
                                String customMessage, String ignoreImageString) throws IOException {
+        boolean modified;
         try (InputStream stream = content.read();
              InputStreamReader streamR = new InputStreamReader(stream);
              BufferedReader reader = new BufferedReader(streamR)) {
-            findImagesAndFix(content, branch, img, tag, customMessage, reader, ignoreImageString);
+             modified = findImagesAndFix(content, branch, img, tag, customMessage, reader,
+                    ignoreImageString);
```

Comment:
Why do you need the `modified` variable?  The return can be within the try-with-resources correct?